cmake-toolchains/linux-aarch64-glibc.cmake

```diff
@@ -0,0 +1,34 @@
+set(CMAKE_SYSTEM_NAME Linux)
+
+# Set CMake variables that subsequent CMake scripts can check against
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+set(TARGET_TRIPLE aarch64-linux-gnu)
+
+# Currently only use clang as it makes cross-compilation easier
+set(CMAKE_ASM_COMPILER_TARGET ${TARGET_TRIPLE})
```

Comment:
You don't have any ASM in this codebase, right? I guess it doesn't really hurt to have this here either way, though.

Reply:
Not right now, but that was the thought. Just to prevent issues if we do in the future and align with Symcrypt-OpenSSL.